src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/WasmPublishTest.cs

```diff
@@ -34,7 +34,8 @@ public void Publish_MinimalApp_Works()
             var testInstance = CreateAspNetSdkTestAsset(testAppName);
 
             var publishCommand = new PublishCommand(Log, testInstance.TestRoot);
-            publishCommand.Execute().Should().Pass();
+            publishCommand.Execute().Should().Pass()
+                .And.NotHaveStdOutContaining("warning IL");
 
             var publishDirectory = publishCommand.GetOutputDirectory(DefaultTfm);
```

Comment:
Which tfm is this?

Reply:
Should be net6. It comes from https://github.com/dotnet/sdk/blob/e50d5ae22ea631d7373713175160cdd16f8a637a/Directory.Build.props#L11

which gets set as metadata on the test assembly in https://github.com/dotnet/sdk/blob/a30e465a2e2ea4e2550f319a2dc088daaafe5649/src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests.csproj#L21-L24
and this is picked up in the test infra:
https://github.com/dotnet/sdk/blob/23e7fd3c08395b368f7db8bee30c20c558b29bd4/src/Tests/Microsoft.NET.TestFramework/AspNetSdkTest.cs#L23

FWIW I did check that the test produces warnings if I set SuppressTrimAnalysisWarnings=false and this causes it to fail.